osquery/tables/yara/yara.cpp

```diff
@@ -38,6 +38,11 @@ FLAG(bool,
      true,
      "Call malloc_trim() after YARA scans (linux)");
 #endif
+FLAG(uint32,
+     yara_intrafile_sleep,
```

Comment:
nitpick, curious why deviate from the convention of `*_delay`?

Reply:
I try not to overthink minor details ... sleep was the first thing that popped into mind.  Probably because the underlying call is `std::this_thread::sleep_for`.  I have no preference either way.